lib/audited/auditor.rb

```diff
@@ -219,9 +219,12 @@ def audit_destroy
       end
 
       def write_audit(attrs)
-        attrs[:associated] = send(audit_associated_with) unless audit_associated_with.nil?
-        self.audit_comment = nil
-        run_callbacks(:audit)  { audits.create(attrs) } if auditing_enabled
+        audits_associated = audit_associated_with.is_a?(Array) ? audit_associated_with : [audit_associated_with]
+        audits_associated.each do |audit_associated|
```

Comment:
Won't this create multiple, duplicate audits, one for each associated object? Though it's more complex, I think it'd make more sense for one audit to exist per event, but to change to the schema to have multiple associations.